adapters/feishu/src/index.ts

```diff
@@ -0,0 +1,12 @@
+import * as Feishu from './types'
+import { FeishuBot } from './bot'
+
+export { Feishu }
+
+export default FeishuBot
+
+declare module '@satorijs/core' {
+  interface Session {
+    Feishu: Feishu.Internal
```

Comment:
这里是不是应该改为小写的 `feishu`, 以及 `adaptSession` 中好像并没有给这个属性赋值，是否也应该把 `Feishu.Event` 也加进去

```suggestion
    feishu: Feishu.Internal & Feishu.Event
```

Reply:
是的，這裡應該是小寫。

這裡加入 Feishu.Event 的目的是什麼？

Reply:
因为用户可能会希望处理一些别的事件，你得把事件的详细内容也加到 session 里。可以参考别的 adapter 也是这样做的，比如 onebot 中加入的是 Payload

Reply:
如此的話，何不在 `interface Event {}` 中加入自定義事件，如 `feishu/interactive` 為了活動卡片。

Reply:
主要是考虑到别的 adapter 也是这样设计的。而且 session 本来就会进入事件，在 `dispatch` 的时候事件就会发出去，所以在加入 `& Event` 的同时，也应该在 `interface Event {}` 中加入事件

https://github.com/satorijs/satori/blob/a3ba417bdc99f5df9cfda6a9b1b0f7a1d79b7095/packages/core/src/bot.ts#L94-L106